ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -534,6 +534,19 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Should this not have the R roles still for judiciary, gatekeepers and magistrates?

Reply:
Removed R Roles for judiciary, gatekeepers and magistrates